src/harvester/ckan.rs

```diff
@@ -107,11 +197,24 @@ struct PackageShow {
     result: Package,
 }
 
+#[derive(Deserialize)]
+struct PackageSearch {
+    success: bool,
+    error: Option<Error>,
+    result: PackageSearchResult,
+}
+
+#[derive(Deserialize)]
+struct PackageSearchResult {
+    count: usize,
+    results: Vec<Package>,
+}
+
 #[derive(Deserialize)]
 struct Package {
     id: Uuid,
     title: String,
-    notes: String,
+    notes: Option<String>,
```

Comment:
This was the last missing dataset at opendata.leipzig.de as it had `notes: null` instead of `notes: ""`.